auth-table-based-service/src/main/java/io/stargate/auth/table/AuthnTableBasedService.java

```diff
@@ -51,6 +55,9 @@
   private static final boolean shouldInitializeAuthKeyspace =
       Boolean.parseBoolean(System.getProperty("stargate.auth_tablebased_init", "true"));
 
+  private final Cache<Object, Object> tokenCache =
```

Comment:
* Why are we not having properly typed cache?
* Why are we recording stats here?

Reply:
Fixed in 4e0e19c68b3adf98849671dae4c498a3ffa02c44